remember.css

```diff
@@ -35,64 +35,77 @@ table {border-collapse: collapse; border-spacing: 0;}
 
 .container {
 	max-width: 1440px;
-	margin: 20px 50px;
+	margin: 22px 70px;
 }
 
-header {
+.header {
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
 }
 
-.item1 {
-	height: 50px;
-	width: 200px;
-	background-color: grey;
+.header__item {
+	height: 84px;
+	width: 300px;
+	background-color: #C4C4C4;
 }
 
-article {
-	margin-top: 50px;
+.small {
+	height: 35px; 
+	width: 50px;
+}
+
+.article {
+	margin-top: 140px;
 	display: grid;
-	grid-row-gap: 40px;
-	grid-column-gap: 20px;
-	grid-template-columns: repeat(3, 1fr);
-	grid-template-rows: repeat(3, 300px);
+	grid-row-gap: 60px;
+	grid-column-gap: 45px;
+	grid-template-columns: 450px 1fr 1fr;
+	grid-template-rows: repeat(3, 500px);
 }
 
 .purple {
- 	background-color: purple;
+ 	background-color: #C978C6;
 	grid-column-start: 1;
 	grid-column-end: 4;
 	grid-row-start: 2;
 }
 
 .green {
-	background-color: green;
+	background-color: #35B451;
 }
 
 .blue {
-	background-color: blue;
+	background-color: #5E70B4;
 	grid-column-start: 2;
 	grid-column-end: 4;
 }
```

Comment:
When you use such classes as blue, purple etc... they suppose to contain some styles connected with the color (based on the name of the class. So when I see a class named "blue", I expect that its styles will contain only some color or background stuff). 
However here you use these classes not only for color but also for some positioning.